### DISCUSSION TITLE
Get last release from file instead of git tag

### DISCUSSION THREAD

**Discussion by galindro** [semantic-release/semantic-release#2029] [Q&A] [answered]
Is it possible to somehow configure semantic-release to get the last release from a file within the repository instead of a git tag?

I'm asking this because I have a Julia repository with many packages. Each package has it's own version. Therefore, I can't control the version by git commit tags.

I would version each package by adopting a commit message convention like this:

```
<type>(packageN): description
```

This way, I know which package is being changed and I could use [this plugin](https://github.com/google/semantic-release-replace-plugin) in order to update the Project.toml accordingly. 

But, semantic-release needs to get the last release from the package's Package.toml, not from git tag

**Comment by travi** [comment, marked as answer]
semantic-release does not officially support monorepos. it expects there to be a 1:1 relationship between the repository and the library being released.

if you are wanting to use semantic-release in a monorepo situation like you describe, you will need to look into community provided tools like [semantic-release-monorepo](https://github.com/pmowrer/semantic-release-monorepo), but those are not supported by this team.

**Reply by galindro** [reply]
Ty very much for your prompt answer. Is this feasible? https://github.com/semantic-release/semantic-release/issues/193#issuecomment-880788733
